mercuryseries/inertia-bundle/0.1/config/packages/mercuryseries_inertia.yaml

```diff
@@ -0,0 +1,38 @@
+mercuryseries_inertia:
+    # |--------------------------------------------------------------------------
+    # | Server-side Rendering
+    # |--------------------------------------------------------------------------
+    # |
+    # | These options configures if and how Inertia uses Server Side Rendering
+    # | to pre-render the initial visits made to your application's pages.
+    # |
+    # | You can specify a custom SSR bundle path, or omit it to let Inertia
+    # | try and automatically detect it for you.
+    # |
+    # | Do note that enabling these options will NOT automatically make SSR work,
+    # | as a separate rendering service needs to be available. To learn more,
+    # | please visit https://inertiajs.com/server-side-rendering
+    ssr:
+        enabled: false
+        url: 'http://127.0.0.1:13714/render'
+        bundle: '%kernel.project_dir%/public/build-ssr/ssr.js'
```

Comment:
Where does this file come from? Does the bundle make this available? Or is it something the user should add?

Reply:
The user needs to execute the `npm run build` command in order to generate the file. The presence of this file is already managed on the bundle side: https://github.com/mercuryseries/inertia-bundle/blob/main/src/Command/StartInertiaSsrCommand.php#L46-L59.